src/main/java/frc/robot/subsystems/arm/ArmIOTalonFX.java

```diff
@@ -141,7 +145,31 @@ public void updateInputs(ArmIOInputs inputs) {
   }
 
   public void setPosition(double position) {
-    arm.setControl(mmv.withPosition(position));
+    // need to check the target position
+    boolean withinAcceptedRange = NemesisMathUtil.isBetweenInclusive(
+      position, 
+      Constants.ArmConstantsLoki.ELEVATOR_FACTORY_MIN_POS, 
+      Constants.ArmConstantsLoki.ELEVATOR_FACTORY_MAX_POS
+    );
+
+    /**
+     * if (its not between accepted setpoints){
+     * if (elevator isn't in the right position){
+     * sout(can't move)}
+     * } else{
+     * then you can move }
+     * 
+     * if (not within accepted setpoints and elevator isn't right )
+     * 
+     * if (within accepted or elevator is right )
+     */
+
+     if (withinAcceptedRange || checkElevator()){
+      arm.setControl(mmv.withPosition(position));
+     }
+     else{
+      System.out.println("CAN'T MOVE ARM, elevator not in valid position. ");
```

Comment:
Fine for testing / debugging.  But if we think we want to know about this condition happening once in competitions, etc, we need to make it more visible than just with a console log.  ie. shuffleboard message, etc.